schemas/theme.json

```diff
@@ -122,6 +122,18 @@
 			},
 			"additionalProperties": false
 		},
+		"notify": {
+			"type": "object",
+			"properties": {
+				"title_info": { "type": "string" },
+				"title_warn": { "type": "string" },
+				"title_error": { "type": "string" },
+				"icon_info": { "$ref": "#/$defs/style" },
+				"icon_warn": { "$ref": "#/$defs/style" },
+				"icon_error": { "$ref": "#/$defs/style" }
```

Comment:
By the looks of it I think you mixed up the `title_` and `icon_` properties. The `title_` properties should use the `style` definitions and the `icon_` properties should be strings right?

Reply:
I am not sure. I just copy them from sxyazi/yazi#717 this issue.

Reply:
Oh I messed it up, it's indeed my fault.

Reply:
Huh? You tell me!